index.html

```diff
@@ -123,17 +143,79 @@
     // SCALES
 
     x = d3.scale.linear()
-      .domain([0, d3.max(data, function (d) { return d.unemployment; })])
+      .domain(d3.extent(data, function (a) {return a.costs; }))
       .range([0, width])
       .nice();
 
     y = d3.scale.linear()
-      .domain(d3.extent(data, function (d) { return d.inflation; }))
+      .domain(d3.extent(data, function (a) {return a.visits; }))
```

Comment:
Good. I'd prefer to use `[0, d3.max(...)]` here in order to make clear from just reading your code that you expect your Y axis to start at 0. Maybe that's not what you intend, though, and just a result of the handful of points that are zero.